styles/onyx/components/table/_variables.scss

```diff
@@ -41,4 +41,13 @@ $et-indent-amount: $spacing-double;
 $et-selectable-row-hover-background: $background-hover;
 $et-selectable-row-selected-background: $background-hover-active;
 
-
+// Editable Table
+// -----------------------------------------------------------------------------
+$editable-table-header-bg-color: $background-extra-light;
+// TODO: Refer back to core-variables when shadow section is no longer broken
+$editable-table-header-box-shadow: 4px 2px 8px 0px rgba(0, 0, 0, 0.1);
```

Comment:
any particular reason why we're offsetting the shadow vertically and horizontally? I believe design arrived at shadows only being offset vertically.

Reply:
A typical one sided drop shadow doesn't fully reach the sides, and causes gaps between the columns like this. 
<img width="573" alt="screen shot 2018-08-23 at 10 25 04 am" src="https://user-images.githubusercontent.com/12617682/44541547-28e71f80-a6bf-11e8-998f-9eb7acf14020.png">

The adjusted it to be full width so it appears as one shadow across the columns, but that means it spills over into the previous column on the sides like this. 
<img width="574" alt="screen shot 2018-08-23 at 10 28 47 am" src="https://user-images.githubusercontent.com/12617682/44541611-5d5adb80-a6bf-11e8-9e1d-1222ee29dfec.png">

The offset shifts it over, and relies on the next `th` to cover the other side. Which is not great, but since box shadow on the `tr` or `thead` is not an option, this seems to be simplest option that works. (old table was just a bunch of divs so it was much easier). I thought about adding inset shadow to first row in the body, but the table can scroll underneath the head, so not an option. Tried pseudo element tricks, but just turned out to be way hackier than this.

Reply:
Got it, sounds good. 

LGTM